app/views/templates/spinner.scala.html

```diff
@@ -0,0 +1,18 @@
+@*
```

Comment:
Do we need both this and app/views/helpers/spinner.scala.html?

Reply:
Will delete one of them

Reply:
Fixed